lib/Elevate/Components/AbsoluteSymlinks.pm

```diff
@@ -0,0 +1,49 @@
+package Elevate::Components::AbsoluteSymlinks;
+
+=encoding utf-8
+
+=head1 NAME
+
+Elevate::Components::AbsoluteSymlinks
+
+Alter absolute symlinks in / to be relative links
+
+=cut
+
+use cPstrict;
+
+use Cpanel::Chdir ();
+use Cpanel::UUID  ();
+use File::Copy    ();
+
+use parent qw{Elevate::Components::Base};
+
+sub get_abs_symlinks {
+    my %links;
+    foreach my $entry (glob "/*") {
+        my $path = readlink($entry); # don't bother with stat, this is fast
+        next unless $path && substr( $path, 0, 1 ) eq '/';
+        $links{$entry} = $path;
+    }
+    return %links;
+}
+
+sub pre_leapp {
+    my %links = get_abs_symlinks();
+    return unless %links;
+    my $chdir = Cpanel::Chdir->new("/");
+    foreach my $link (keys(%links)) {
+        my $updated = substr( $links{$link}, 1 );
+
+        my $rand_uid = Cpanel::UUID::random_uuid();
```

Comment:
note: even if the probability of collision is low, it can conflict...

Reply:
Conflict with what? ideally nothing is happening on the system when they're upgrading?

Reply:
yea... I realize nothing is ever truly "random" ever either... I'm just trying to *avoid* failures here when possible. Collision chances here should be so vanishingly small as to not ever be hit in the wild.